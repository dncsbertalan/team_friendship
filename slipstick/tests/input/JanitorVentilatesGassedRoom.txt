load testmap.txt
move Janitor_1 Room_9
move Janitor_1 Room_10
state -r Janitor_1
exit