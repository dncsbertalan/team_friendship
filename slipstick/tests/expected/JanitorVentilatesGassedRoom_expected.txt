Successfully loaded testmap.txt
Moving Janitor_1 from Janitors'Room to Room_9
Moving Janitor_1 from Room_9 to Room_10
Room_4 state: cleaned